### PR TITLE
STREAM-16 Convert over to new serialization framework.

### PR DESCRIPTION
**Pull request by akuma432**
* Added lombok dependency to generate Equals and hashcode method for comparing two java objects.
* Added Junit test case to deserialise and serialise config files if source type is Jdbc or Glue and destination type is snowflake.
* Added Config files for Junit if source type is Jdbc or Glue and destination type is snowflake.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -40,7 +40,8 @@ lazy val root = (project in file("."))
       "org.scalatest" %% "scalatest" % "3.0.5" % "it,test",
       "junit" % "junit" % "4.13.2" % "it,test",
       "org.testcontainers" % "mysql" % "1.15.1" % "it,test",
-      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.12.3"
+      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.12.3",
+      "org.projectlombok" % "lombok" % "1.18.20" % "provided"
```
Comment (brockn) [inline build.sbt:44]: Since we are adding lombok, perhaps we can use @Getter/@Setter in the POJO classes and delete the ones we generated.

**Comment by brockn** [comment]
Looks good!

File: build.sbt (hunk 1 above)
Comment (akuma432) [reply build.sbt:44]: deleted all generated getter and setter method

**Comment by brockn** [comment]
Approved

**Pull request merged**